test/java/pages/NewPageEditor.java

```diff
@@ -0,0 +1,27 @@
+package pages;
+
+import abstractPackage.BasePageClass;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class NewPageEditor extends BasePageClass {
+
+    By pagesTitle = By.id("title");
```

Comment:
1. naming - constants in capital letters
2. constant variables as private static final